apps/crn-server/src/handlers/working-group/update-deliverables-handler.ts

```diff
@@ -0,0 +1,97 @@
+import 'source-map-support/register';
+
+import { Handler } from 'aws-lambda/handler';
+import { EventBridgeHandler } from '@asap-hub/server-common';
+import {
+  RestWorkingGroup,
+  SquidexGraphql,
+  SquidexRest,
+} from '@asap-hub/squidex';
+import { WorkingGroupDeliverable, DeliverableStatus } from '@asap-hub/model';
+import logger from '../../utils/logger';
+import { sentryWrapper } from '../../utils/sentry-wrapper';
+import { WorkingGroupEvent, WorkingGroupPayload } from '../event-bus';
+import {
+  WorkingGroupDataProvider,
+  WorkingGroupSquidexDataProvider,
+} from '../../data-providers/working-groups.data-provider';
+import { getAuthToken } from '../../utils/auth';
+import { appName, baseUrl } from '../../config';
+
+type PartialMapping<T extends string> = { [S in T]?: T };
+
+export const workingGroupUpdateHandler =
+  (
+    dataProvider: WorkingGroupDataProvider,
+  ): EventBridgeHandler<WorkingGroupEvent, WorkingGroupPayload> =>
+  async (event) => {
+    logger.info(
+      `Received ${event.detail.type} event for ${event.detail.payload.data?.title?.iv}`,
+    );
+
+    const {
+      detail: {
+        payload: { id },
+      },
+    } = event;
+    const workingGroup = await dataProvider.fetchById(id);
+
+    if (!workingGroup) {
+      return;
```

Comment:
Throwing would be fine too, but I don't _think_ it matters since this is being called in response to an EventBridge event, so there's no client dependent on a particular response. It's mostly just there because the type checker insisted it might not exist.

I guess the benefit of throwing would be that it would get logged/alerted somewhere?

Reply:
It looks like there's a modest benefit to a throw because it means AWS will retry the invocation, which if the failure is due to transient network/infa issues will be preferable IMO.

Have replaced.